README.md

```diff
@@ -128,12 +125,96 @@ Invokes all Unit tests from schema/package "hr_test" with two reporters:
 * ut_coverage_html_reporter - will report only on database objects that are mapping to file structure from "source" folder and save output to file "coverage.html"
 
 ```
-utplsql run hr/hr@xe
+> utplsql run hr/hr@xe
 ```
 
 Invokes all unit test suites from schema "hr". Results are displayed to screen using default ut_documentation_reporter.
 
-#### Enabling Color Outputs on Windows
+### info
+`utplsql info [<ConnectionURL>]`
+
+```
+<ConnectionURL>     - accepted formats:
+                        <user>/<password>@//<host>[:<port>]/<service>
+                        <user>/<password>@<host>:<port>:<SID> 
+                        <user>/<password>@<TNSName> 
+                      To connect using TNS, you need to have the ORACLE_HOME environment variable set.
+                      The file tnsnames.ora must exist in path %ORACLE_HOME%/network/admin
+                      The file tnsnames.ora must contain valid TNS entries. 
+```
+
+#### Examples
+
+```
+> utplsql info
+
+cli 3.1.1-SNAPSHOT.local
+utPLSQL-java-api 3.1.1-SNAPSHOT.123
+```
+```
+> utplsql info app/app@localhost:1521/ORCLPDB1
+
+cli 3.1.1-SNAPSHOT.local
+utPLSQL-java-api 3.1.1-SNAPSHOT.123
+utPLSQL 3.1.2.1913
+```
+
+### reporters
+`utplsql info <ConnectionURL>`
```

Comment:
Is it `info` or `reporters` ?
The examples show different command.

Reply:
It was definitely too late :) Thanks for finding - it's `reporters`